shell/src/test/resources/log4j.properties

```diff
@@ -1,17 +0,0 @@
-tachyon.root.logger=INFO, TEST_LOGGER
```

Comment:
Now there's no shell tests, since they're all in integration tests now. This was the only file left over.